### ISSUE TITLE
ME Interface Direct Connect Fails

### ISSUE THREAD

**Issue by Poisonsting** [DarkholmeTenk/TardisModIssues#492]
When using a Sonic Screwdriver in Linking mode on 2 different Roundels a chat entry stating the two are linked appears, but at least 1 of the roundels always reports Direct Connect: None on right click with an empty hand afterward, and all cables/devices plugged into any Roundel connect directly to the 32 channel limited internal network.

**Comment by AlienXtream** [comment]
the linking for AE networks is... unintuitive. do you have a security matrix installed in the AE network?

**Comment by Poisonsting** [comment]
I saw another bug where the security terminal was an issue, so I removed to test. There was no difference. For reference I was running the release version of the Tardis mod, and I'm now running the latest from Jenkins because of another bug, neither versions work.

**Comment by AlienXtream** [comment]
the issue with the security matrix is that basically the tardis doesn't have permission to edit the network so a blank biometric card (sets default permissions) fixes that one. try linking from the outward channels (from the controller) to receiver networks (machines etc...). if you fiddle around with it a bit you should start to notice a pattern (far to hard to explain). in general the method dark uses for linking tardis ME interfaces is weird. i had various issues that i could not track down or explain (as if the network was updating in the wrong order). AE is VERY complex when you have more that 32 channels so trying to distinguish between user error, AE bug and TM bug is.... challenging. if i can come up with another system then i will try and talk dark into reworking it ;P (probs not before 1.9 though)

**Comment by Poisonsting** [comment]
I appreciate how intricate both mods are, and I'm really thankful to you guys for even making it in the first place. I know you've likely tried pretty much every obvious solution, and I imagine you're as frustrated as any user would be, but I just wish it were possible to make the roundels work like p2p Tunnels.

Please keep me posted if you guys try anything new, no matter how long that takes.

**Comment by AlienXtream** [comment]
then why not have a closed sub network with nested P2P channels. that way, in theory, you can have an infinite number of channels without using dense cable.

**Comment by Poisonsting** [comment]
Well, you're still limited to 32 p2p Tunnel exit points within the Tardis, but you can have an infinite number of channels per location.

**Comment by AlienXtream** [comment]
nested p2p tunnels. have a p2p network carry a p2p network carrying a p2p network. it stacks theoretically infinitely (or until your game/server can't handle it anymore)

**Comment by Poisonsting** [comment]
Yes, but the Tardis itself only has 32 channels. And to transport the P2P signal through roundels you still need a channel. Once you get the P2P Tunnel to the location in the Tardis, then you can go infinite. But there are a maximum of 32 different places that can happen.
